cmd/config/config.go

```diff
@@ -16,16 +16,20 @@ var (
 	section       = config.MustRegisterSection("root", defaultConfig)
 )
 
+// Config hold configration for flytectl flag
 type Config struct {
 	Project string `json:"project" pflag:",Specifies the project to work on."`
 	Domain  string `json:"domain" pflag:",Specified the domain to work on."`
 	Output  string `json:"output" pflag:",Specified the output type."`
+	Filters  string `json:"filters" pflag:",Specified the flyte filter i.e. workflow/execution"`
```

Comment:
shouldnt this be a slice?
[]string?
or actually
map[string]string

Reply:
yes